doc/lpqcqp.rst

```diff
@@ -93,6 +93,13 @@ to indicate equality constraints.
     objective coefficient ``objcoef``. This is equivalent to calling the
     above method with empty arrays for the constraint coefficients.
 
+.. function:: delvars!(m::AbstractLinearQuadraticModel, idxs)
+
+    Removes the variables with indexes in the vector idxs. The remaining variables in the model
+    are renumbered. Let, for example, before deletion there be six
+    variables p, q, r, s, t, u with ordinal numbers 1, 2, 3, 4, 5, 6, and let variables
+    p, q, s have been deleted. Then after deletion the remaining variables r, t, u
+    are assigned new ordinal numbers 1, 2, 3.
```

Comment:
The wording here is confusing. There's no concept of symbolic names for variables in MPB. I would say something like, "suppose we delete indices 1, 2, and 4, then there will be 3 variables remaining in the model, those which were previously numbered 3, 5, and 6."